docs/release-notes/index.rst

```diff
@@ -60,6 +60,8 @@ development keeps pace with the OS's new releases and support lifecycle.
 .. toctree::
    :maxdepth: 1
 
+   rockcraft-1-7-0
```

Comment:
Without this entry, the build fails because the document isn't added in any table of contents. However, *with* this entry this release-notes doc ends up having two listings for a given version:

![image](https://github.com/user-attachments/assets/e8989bf3-53fa-4f02-b191-56e52da8a157)


Should this toctree be `:hidden:`?

Reply:
> Should this toctree be :hidden:?

Absolutely.